ansible/workers.yml

```diff
@@ -131,11 +132,18 @@
 
   # this is the only common way across rhel/fedora.
   # It's url-based, because original project is dead, and fork is not established yet
-  - name: Ensure linkchecker is installed for SCAP Security Guide checking 
+  - name: Ensure linkchecker is installed for SCAP Security Guide checking
     pip:
       name:
       - git+https://github.com/linkcheck/linkchecker.git
 
+  # RHEL may not have pytest
```

Comment:
may not? does it have it or not?

CentOS7 seems to have it:
https://centos.pkgs.org/7/centos-x86_64/pytest-2.7.0-1.el7.noarch.rpm.html

and it is in EPEL6.

Please use the packages instead of pip.

Reply:
RHEL6 doesn't have it, but RHEL7 does.